torch/_dynamo/variables/nn_module.py

```diff
@@ -1012,9 +1015,16 @@ def call_function(
             else nullcontext()
         )
         with ctx:
-            return variables.UserFunctionVariable(fn, source=source).call_function(
-                tx, [self] + list(args), kwargs
-            )
+            if isinstance(fn, functools.partial):
```

Comment:
Could you handle this in VariableBuilder instead? I think we've been moving away from carveouts like this and want to consolidate in VariableBuilder.